product_docs/docs/efm/4/04_configuring_efm/05_using_vip_addresses.mdx

```diff
@@ -103,23 +133,26 @@ When instructed to ping the VIP from a node, use the command defined by the `pin
 
 3. Ping the VIP from the other nodes to verify that they can reach the VIP:
 
-   ```text
-   # /bin/ping -q -c3 -w5 172.24.38.239
+   ```console
+   /bin/ping -q -c3 -w5 172.24.38.239
+   __OUTPUT__
    PING 172.24.38.239 (172.24.38.239) 56(84) bytes of data.
    --- 172.24.38.239 ping statistics ---
    3 packets transmitted, 3 received, 0% packet loss, time 1999ms
    rtt min/avg/max/mdev = 0.023/0.025/0.029/0.006 ms
    ```
 
-   No packet loss occurs.
-   !!!Important
-      Failover Manager uses the exit code of the ping command to determine whether the address was reachable. In this case, the exit code is zero. If you're using a command other than ping, it must return a zero exit code if the address is reachable.
+   You will see 0% packet loss, indicating the IP now reaches the machine configured as the primary node.
+
+   !!!tip "Meaning of the ping command exit code for reachable addresses"
+      Failover Manager uses the exit code of the ping command to determine whether the address was reachable. In this case, the exit code is zero. If you're using a command other than ping, it must return a zero exit code when the address is reachable.
 
-4. Use the `efm_address del` command to release the address on the primary node and confirm the node was released with ip address:
+4. Use the `efm_address del` command to release the address on the primary node and confirm the VIP was released with the `ip address` command:
 
-   ```text
-   # efm_address del eth0 172.24.38.239/24
-   # ip address
+   ```console
+   efm_address del eth0 172.24.38.239/24
+   ip address
+   __OUTPUT__
    eth0 Link encap:Ethernet HWaddr 22:00:0A:89:02:8E
```

Comment:
Right after this there is some old text that says "The output from this step doesn't show an eth0 interface." Can you fix that? The interface is still there, just not the virtual IP address.

Reply:
👍